indexer/api/handlers/common.go

```diff
@@ -0,0 +1,51 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// lazily typing numbers fixme
+type Transaction struct {
```

Comment:
We can iterate on this but similar to `database.Transaction` i think this should follow tx convention, `From, To, Value, ..`

Things like BlokcHash, BlockNumber, Timestamp i think should live top level to the Deposit/Withdrawal.